libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -186,6 +186,7 @@
         "RSM017_OUTBOUND": "Anmod om engrosydelser (RSM-017)",
         "RSM019_INBOUND": "Fremsend beregnede engrosydelser (RSM-019)"
       },
+      "statusPlaceholder": "Status",
```

Comment:
Suggestion: Looks like you are missing some translations
![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1250254/1495fa1a-73b0-4ca2-b95d-3aeb2587a63f)

Reply:
Good catch. Was missing an import.